uncrustify.cfg

```diff
@@ -269,6 +269,9 @@ nl_for_brace                            = remove        # string (add/force/igno
  
 # Newline between function call and open brace
 nl_fcall_brace                          = remove        # string (add/force/ignore/remove)
+
+# Newline between objective-c method and open brace
+nl_oc_mdef_brace						= remove        # string (add/force/ignore/remove)
```

Comment:
It seems that you have different number of spaces, right? :)

Reply:
Resolved https://github.com/CocoaLumberjack/CocoaLumberjack/pull/1026/commits/19103c9c5c4991bc1a3473eac010e3af7071c6b6